js/_codemirror.ts

```diff
@@ -50,11 +50,11 @@ const asmErrorTooltip = {
     'color':      'var(--background)',
 };
 
-const fontFamily = "'Source Code Pro', monospace";
+const fontFamily = '\'Source Code Pro\', monospace';
```

Comment:
I find this harder to read, not sure why eslint doesn't flag it with out current rule of `quotes: [ error, single ]`, perhaps we just need a `"avoidEscape": true`? https://eslint.org/docs/rules/quotes